lib/cmps/sourcepoint-frame.ts

```diff
@@ -7,10 +7,11 @@ import AutoConsentCMPBase from "./base";
 export default class SourcePoint extends AutoConsentCMPBase {
   prehideSelectors = ["div[id^='sp_message_container_'],.message-overlay"]
 
-  ccpaMode = false;
+  ccpaNotice = false;
+  ccpaPopup = false;
 
   runContext: RunContext = {
-    main: false,
+    main: true,
```

Comment:
Does this popup run in the main frame? Where I can test it?

Reply:
This is to make the prehide selectors work: https://app.asana.com/0/0/1203411040749543/1203463729387875/f

Reply:
Prehide selectors need to be applied in the top frame. But now I realize that they don't depend on the runContext, so this is indeed not necessary. Thanks for catching this!